app/services/pro_memberships/biller.rb

```diff
@@ -0,0 +1,94 @@
+module ProMemberships
+  # Bills pro memberships that expire today
+  class Biller
```

Comment:
This is going to be called daily and we renew all memberships that have the date of today, regardless of hour.